.github/workflows/test-e2e.yml

```
@@ -8,7 +8,7 @@ on:
 
 jobs:
   test-e2e:
-    name: Run on Ubuntu
+    name: E2E Tests
```

```suggestion
    name: E2E (Kind)
```
AND (can't make a suggestion if line wasn't edited in this PR) change L1 of this file:
change `name: E2E Tests` → `name: Tests`

As a result workflow step would look as `Tests / E2E (Kind)` in PR / CI.